Dalamud/Interface/Internal/Windows/PluginInstaller/PluginInstallerWindow.cs

```diff
@@ -2556,7 +2556,7 @@ private void DrawInstalledPluginContextMenu(LocalPlugin plugin, PluginTestingOpt
 
             if (ImGui.MenuItem(Locs.PluginContext_DeletePluginConfigReload))
             {
-                this.ShowDeletePluginConfigWarningModal(plugin.Name).ContinueWith(t =>
+                this.ShowDeletePluginConfigWarningModal(plugin.Manifest.InternalName).ContinueWith(t =>
```

Comment:
Doesn't this show the InternalName in the pop-up rather than the actual plugin name?

Reply:
Yes, the regular name seemed kinda weird because for chat2 it showed "vChat 2"
And with internal name it showed "vChatTwo"

I'm fine with reverting that line if you prefer.

Reply:
Please do. We don't expose the internal name anywhere else, and some plugins (Allagan Tools) have markedly different InternalNames that may confuse users.

If you want to fix the stray `v`, `PluginInstallerWindow.cs` L3710